robottelo/ui/user.py

```diff
@@ -91,11 +91,18 @@ def create(self, username=None, email=None, password1=None,
                 )
         self.click(common_locators['submit'])
 
-    def search(self, name, search_key):
+    def search(self, element_name, element_locator=None, search_key=None,
```

Comment:
As we are re-factoring things, how about we set search_key='login' by default here?

We can just get rid of handling this stuff for all the tests. thoughts?